src/interface/api-swagger-iso20022-parties.yaml

```diff
@@ -890,13 +890,14 @@ components:
   schemas:
     CorrelationId:
       title: CorrelationId
-      pattern: ^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$
       type: string
-      description: Identifier that correlates all messages of the same sequence. The
-        API data type UUID (Universally Unique Identifier) is a JSON String in canonical
-        format, conforming to RFC 4122, that is restricted by a regular expression
-        for interoperability reasons. An UUID is always 36 characters long, 32 hexadecimal
-        symbols and 4 dashes (‘-‘).
+      pattern: ^[0-9a-f]{8}-[0-9a-f]{4}-[1-7][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$|^[0-9A-HJKMNP-TV-Z]{26}$
```

Comment:
_api-snippets_ already has this change:
https://github.com/mojaloop/api-snippets/blob/main/fspiop/v2_0/openapi3/components/schemas/CorrelationId.yaml#L3